### ISSUE TITLE
[question] where is app directory on android devices?

### ISSUE THREAD

**Issue by arooni** [koreader/koreader#4585]
* KOReader version: 2019.02 stable release
* Device: kindle fire 10 tablet

#### Issue
I wanted to copy over all settings (looks like this might be supported by zsync, NOT SURE), but also the dictionaries on kindle that I didn't just download thru koreader.  

When I went to look in Android/data/ I didn't see anything that indicated the koreader.  So not sure where the koreader files is on android, or if there is even one.  Would be handy to know to put dictionaries/etc.  

TLDR:
1) Where is the KOreader folder on android devices?
2) is there already a way to sync all settings from one koreader device to another so I don't have to duplicate the setup process on any new device (my tablet, my phone) after setting up kindle.  thanks!
3) are note/marks that i highlight synchronized to other devices?  [update: it appears not]  is this a planned feature?  or should I open a FR?

Thanks!

**Comment by poire-z** [comment]
(Can you please cleanup your latest issues - we did it for you on the past ones, hoping you would notice: remove the sample sections that are irrelevant, like the huge `crash.log (if applicable)`. Thank you!)

**Comment by Frenzie** [comment]
In Android the settings are typically stored in a location like `/sdcard/koreader`. (Whether there's an actual sdcard involved is fairly immaterial to Android.)

**Comment by arooni** [comment]
> (Can you please cleanup your latest issues - we did it for you on the past ones, hoping you would notice: remove the sample sections that are irrelevant, like the huge `crash.log (if applicable)`. Thank you!)

absolutely!  thanks for pointing that out.  you folks actually contributing the code are doing the hard part.

**Comment by pazos** [comment]
> Where is the KOreader folder on android devices?

This is device dependant. We use android sdk to retrieve internal and external storage path. Internal path is somewhere in /data/data and is used by libraries and binaries. External path is somewhere on your internal storage (/sdcard, /storage/emulated/0 or whatever). We can print that information somewhere if that is useful.

**Comment by arooni** [comment]
I found it; its on the root directory of /sdcard; what was weird is I didn't see it on the android file transfer tool i use to transfer files on mac

**Comment by Frenzie** [comment]
@pazos But for now `/sdcard` is still always a default symlink to "`/storage/emulated/0` or whatever" afaik.
